geniza/corpus/templates/admin/corpus/document/delete_selected_confirmation.html

```diff
@@ -0,0 +1,6 @@
+{% extends "admin/delete_selected_confirmation.html" %}
```

Comment:
is this one for deleting multiple documents? good catch, thanks for generalizing the message

Reply:
Exactly!